x-pack/plugins/observability/public/pages/cases/use_fetch_alert_data.test.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { act, renderHook } from '@testing-library/react-hooks';
+import { kibanaStartMock } from '../../utils/kibana_react.mock';
+import { useFetchAlertData } from './use_fetch_alert_data';
+
+const mockUseKibanaReturnValue = kibanaStartMock.startContract();
+
+jest.mock('../../utils/kibana_react', () => ({
+  __esModule: true,
```

Comment:
lol what is that?

Reply:
haha found it in one of the other tests in observability 🤷‍♂️ 
https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options

Reply:
This is a way to mock es modules. https://jestjs.io/docs/jest-object#jestmockmodulename-factory-options